util/util.go

```diff
@@ -132,3 +136,26 @@ func IntV(in *int) int {
 	}
 	return *in
 }
+
+// FSGeneration generates a name and a path that will be used to feed
+// the `fsMap` in case consumer provided an empty map.
+// FSMap is required to link the filesystem from v2 to v3 with the rest of the configuration.
+func FSGeneration(name string, fsMap map[string]string) error {
+	if len(name) == 0 {
+		rand.Seed(time.Now().Unix())
+		generatedName := rand.Int()
+		name = strconv.Itoa(generatedName)
+	}
+
+	if _, ok := fsMap[name]; !ok {
+		// generate a random path
+		p, err := ioutil.TempDir("", name)
```

Comment:
When the user gives a name that accidentally is the same as something existing under `/tmp` this here will fail. Should we prevent this by having a random suffix being added?

```suggestion
		p, err := ioutil.TempDir("", name+ "-*")
```